task/mypy-lint/0.1/mypy-lint.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mypy-lint
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: linter
+    tekton.dev/displayName: "Mypy(Python) linter"
+spec:
+  description: >-
+    This task can be used to perform lint check on python files
+  workspaces:
+    - name: shared-workspace
+      description: A workspace that contains the fetched git repository.
+  params:
+    - name: args
+      type: array
+      description: extra args needs to append
```

```suggestion
      description: extra args needs to be appended
```
Maybe we can change this too.